src/main/java/edu/cornell/kfs/concur/batch/service/impl/ConcurSAELineType.java

```diff
@@ -0,0 +1,15 @@
+package edu.cornell.kfs.concur.batch.service.impl;
+
+/**
+ * Helper enum for categorizing various types of SAE lines.
+ */
+public enum ConcurSAELineType {

```

Comment:
Since this appears to be just used in Collector processing, would it be better to call this ConcurSAECollectorLineType, or something so it's name suggest it is just for Collector, and not PDP?

Reply:
Good point, I'll rename this accordingly to avoid confusion with the PDP side.